bower.json

```diff
@@ -12,12 +12,11 @@
     "url": "git://github.com/purescript-contrib/purescript-coroutines.git"
   },
   "dependencies": {
-    "purescript-freet": "^4.0.0",
+    "purescript-freet": "^5.0.0",
```

Comment:
Maybe we could use the range `4 - 5` and keep the `devDependency`. That way this would be a patch change rather than breaking.